src/pages/Merchant/LoginPage/LoginPage.js

```diff
@@ -49,7 +62,7 @@ function LoginPage() {
         <span>100+ products available at best price</span>
         <img
           className={styles.loginLeftImage}
-          src={leftPanelImage}
+          src={authLeftPanelImage}
           alt=""
```

Comment:
anytime anywhere if using image tag always pass alt some values as if the image does not load for any reason alt will show what this image was for